Remotion/Core/Reflection.UnitTests/CodeGeneration/MethodWrapperEmitterTests/SetUpFixture.cs

```diff
@@ -46,13 +46,19 @@ public virtual void OneTimeSetUp ()
       Console.WriteLine ("Setting up MethodWrapperEmitterTests");
 
       var assemblyName = new AssemblyName ("Remotion.Reflection.CodeGeneration.MethodWrapperEmitterTests.Generated.Unsigned");
+      s_assemblyBuilder = AssemblyBuilder.DefineDynamicAssembly (assemblyName, AssemblyBuilderAccess.Run);
```

Comment:
Okay, not sure what happened, but this was the expected code:

```
    [OneTimeSetUp]
    public virtual void OneTimeSetUp ()
    {
      Console.WriteLine ("Setting up MethodWrapperEmitterTests");

      var assemblyName = new AssemblyName ("Remotion.Reflection.CodeGeneration.MethodWrapperEmitterTests.Generated.Unsigned");
      var moduleName = assemblyName + ".dll";
#if NETFRAMEWORK && !NO_PEVERIFY
      s_assemblyBuilder = AppDomain.CurrentDomain.DefineDynamicAssembly (
                assemblyName,
                AssemblyBuilderAccess.RunAndSave,
                TestContext.CurrentContext.TestDirectory);
      s_moduleBuilder = s_assemblyBuilder.DefineDynamicModule (moduleName, emitSymbolInfo: false);
#else
      s_assemblyBuilder = AssemblyBuilder.DefineDynamicAssembly (assemblyName, AssemblyBuilderAccess.Run);
      s_moduleBuilder = s_assemblyBuilder.DefineDynamicModule (moduleName);
#endif

      DeleteIfExists (s_moduleBuilder.FullyQualifiedName);
    }
```

I have only copy&pasted it, so I hope I didn't miss something. Please, consider the code structure primarily and anyhting buggy is likely a bug in the review